src/argilla_server/apis/v1/handlers/info.py

```diff
@@ -14,12 +14,21 @@
 
 from fastapi import APIRouter
 
-from argilla_server._version import __version__
-from argilla_server.schemas.v1.info import Version
+from argilla_server.contexts import info
+from argilla_server.schemas.v1.info import Status, Version
 
 router = APIRouter(tags=["info"])
 
 
 @router.get("/version", response_model=Version)
 async def get_version():
-    return Version(version=__version__)
+    return Version(version=info.argilla_version())
+
+
+@router.get("/status", response_model=Status)
+async def get_status():
+    return Status(
+        version=info.argilla_version(),
+        elasticsearch=info.elasticsearch_status(),
```

Comment:
We could change this to `search_engine` since in the code we have a lot of references to the search engine (which internally is an elasticsearch/openseach instance

Reply:
Also, I think using the SearchEngine for this would be better.

```suggestion
        elasticsearch=info.engine_status(search_engine),
```
And internally:
```python
def engine_status(search_engine):
   return search_engine.client.get_cluster_info() # In the future we can wrap this as a public interface method (.status) or something.